sockeye/inference.py

```diff
@@ -371,6 +372,8 @@ def load_models(context: mx.context.Context,
                                         vocabulary.  Used when logits/softmax are handled separately.
     :param cache_output_layer_w_b: Models cache weights and biases for logit computation as NumPy arrays (used with
                                    restrict lexicon).
+    :param override_dtype: Overrides dtype of encoder and decoder to a different from one, defined in config
```

Comment:
Maybe "Overrides dtype of encoder and decoder defined at training time to a different one."?